docs/installation/aws.md

```diff
@@ -22,13 +22,18 @@ Update the `appsettings.json` file:
         "Region": "us-west-1",
         "Bucket": "foo",
         "AccessKey": "",
-        "SecretKey": ""
+        "SecretKey": "",
+        "ServiceUrl": "",
+        "UseHttp": false,
+        "ForcePathStyle": false
```

Comment:
It'd be less confusing to folks if we separate the AWS S3 and MinIO samples:

```suggestion
```

Reply:
Sound good